lib/memjs/utils.js

```diff
@@ -5,13 +5,20 @@ var header = require('./header');
 var bufferify = function(val) {
   return Buffer.isBuffer(val) ? val : Buffer.from(val);
 };
+exports.bufferify = bufferify;
 
-exports.makeRequestBuffer = function(opcode, key, extras, value, opaque) {
+exports.addToRequestBuffer = function(opcode, key, extras, value, opaque, buf, _bufTargetWriteOffset) {
   key = bufferify(key);
   extras = bufferify(extras);
   value = bufferify(value);
-  var buf = Buffer.alloc(24 + key.length + extras.length + value.length);
-  buf.fill();
+
+  var bufTargetWriteOffset = _bufTargetWriteOffset || 0;
+  var totalBytesWritten = 0;
+  function appendToBuffer(toWriteBuffer) {
```

Comment:
like my nit above; would rename to `copyIntoBuffer`; append can mean both "resize dest and then copy" or "copy to end of dest consumbed bytes"

Reply:
done.